Runtime/Scripts/MeshSyncServer.cs

```diff
@@ -14,16 +15,26 @@ namespace Unity.MeshSync {
 /// <summary>
 /// A component to sync meshes/models editing in DCC tools into Unity in real time.
 /// </summary>
-[ExecuteInEditMode]
-public class MeshSyncServer : BaseMeshSync {
+[ExecuteAlways]
+public partial class MeshSyncServer : BaseMeshSync {
 
     /// <summary>
     /// Callback which will be called after MeshSyncServer receives data and finishes processing it
     /// </summary>
     public ServerMessageCallback OnPostRecvMessageCallback = null;
     
 //----------------------------------------------------------------------------------------------------------------------
-    
+
+#if ENABLE_IL2CPP
+    // When building with IL2CPP, we must use a static delegate
+    private static MeshSyncServer Instance;
+
+    [MonoPInvokeCallback(typeof(Server.MessageHandler))]
+    private static void HandleReceivedMessage(NetworkMessageType type, IntPtr data)
+    {
+        Instance.HandleRecvMessage(type, data);
+    }
+#endif
```

Comment:
Please remove this. I wrote this originally, but we can create a separate PR to tackle the IL2CPP builds

Reply:
Done